mypy/fastparse.py

```diff
@@ -94,7 +94,7 @@ def parse_type_comment(type_comment: str, line: int) -> Type:
 
 def with_line(f: Callable[[Any, T], U]) -> Callable[[Any, T], U]:
     @wraps(f)
-    def wrapper(self, ast):
+    def wrapper(self: Any, ast: Any) -> U:
```

Comment:
There _are_ stubs for typed_ast, actually!  There's a real problem here: not all AST nodes have linenos, but there isn't a common superclass (or ABC) that ensures there's a lineno.